docs/user_stories/topology.md

```diff
@@ -0,0 +1,83 @@
+- Voltage level VL2 has 3 nodes, line LN is connected to node 1, busbar section BBS3 to node 2.
+- Transformer TR is connected to node 8 of voltage level 400 KV and node 3 of voltage level 225 KV.
+- Plain edges represent closed switches.
+- Dashed edges represent opened switches.
+- Green edges will disappear during the bus/breaker topology computation whereas pink edges (like in this case 3<->4) will be retained whatever their position are (see [switches](switch.md)).
+
+The code shows how to create the substation with a node/breaker topology model is fully implemented in the tutorial [topology](https://github.com/powsybl/powsybl-tutorials).
+
+## Bus/breaker view
+
+The following diagram shows computed bus/breaker topology. Compared to node/breaker topology, only remains equipements (GN, LD, TR, LN) and switches flagged as retained (BR3). Equipments are now connected through buses (B1 and B2).
+
+![Bus breaker graph](./images/busBreakerTopology.svg){: width="50%" .center-image}
+
+We can switch to a bus/breaker view on the substation voltage level VL1. VL1 contains 2 buses in the bus/breaker view. The first bus connects nodes 1, 2, 3, 5 and 6, and consequently generator GN and load LD. Note that load LD does not belong to the connected component. The second bus connects nodes 4, 7 and 8. Note that VL1 contains only 1 switch in the bus/breaker view (BR3). Here are the corresponding prints in the tutorial:
+
+````
+Bus: VL1_1
+	 Generators: GN
+Bus: VL1_4
+	 T2W: TR
+Bus: VL1_5
+	 Loads: LD, connected component: null
+Switch: BR3
+Bus: VL2_1
+	 T2W: TR
+````
+
+## Bus view
+
+The following diagram shows the computed bus topology. Compared to bus/breaker topology, there is no switches anymore. Only remains equipements (GN, TR, LN) connected through buses. LD is not connected.
+
+![Bus graph](./images/busTopology.svg){: width="50%" .center-image}
+
+We can switch to a bus view on substation voltage level VL1. VL1 contains 1 buses in the bus view. This bus connects all the equipements of voltage level VL1. Here are the corresponding prints in the tutorial:
+
+````
+Bus:VL1_1
+	 Generators: GN
+	 T2Ws: TR
+Bus:VL2_1
+	 T2Ws: TR
+````
+
+# Building a network in bus/breaker topology model
+
+Note that creating a substation with a bus/breaker topology model is easier. Instead of creating VL1 and VL3 with a node/breaker topology model, we can directly create them in a simpler bus/breaker topology model. It can be very useful when data source only contains bus/branch data link in UCTE-DEF or in CIM-CGMES format sometimes. Warning: in that case the node/breaker view status on voltage level VL1 and VL2 is N/A. Here are the corresponding prints in the tutorial. Note that the load LD is not printed in that case:
+
+````
+bus/breaker network in bus/breaker view:
+Bus: B1
+	 Generators: GN
+Bus: B2
+	 T2Ws: TR
+Switches: BR3
+Bus: B3
+	 T2Ws: TR
+bus/breaker network in bus view:
+Bus:VL1_0
+	 Generators: GN
+	 T2Ws: TR
+Bus:VL2_0
+	 T2Ws: TR
+````
```

Comment:
The user story about the topology does not appear in the navigation panel, like the load flow user story and the capacity calculation tutorial, it is thus harder to find.

Reply:
By the way, should I also create a page on the website for the sensitivity tutorial?

Reply:
Done.